sdks/RepoToolset/tools/Performance.targets

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <Target Name="PerformanceTest" Condition="'$(IsPerformanceTestProject)' == 'true'">
+
+    <MakeDir Directories="$(ArtifactsTestResultsDir)"/>
+    <Message Text="Running performance tests: $(TargetPath)" Importance="high"/>
+
+    <Exec Command='$(DotNetTool) $(TargetPath) $(PerformanceTestParameters)'
```

Comment:
>PerformanceTestParameters [](start = 49, length = 25)

Where do these parameters come from?

Reply:
Are they defined by the project?

---
In reply to: [167686054](https://github.com/dotnet/roslyn-tools/pull/158#discussion_r167686054) [](ancestors = 167686054)